app/assets/stylesheets/application/popups.sass

```diff
@@ -2,7 +2,9 @@
 .modal-body
 	input[type='text']
 		text-align: center
-	textarea, input[type='text']
+	.bootstrap-select
+		width: 85% !important
```

Comment:
`!important` is a huge CSS code smell. It is very rarely necessary, and generally there is a better way to work around it. If it's super necessary, then you probably want a comment on why it's absolutely necessary.